lib/net/ldap.rb

```diff
@@ -309,7 +309,14 @@ class Net::LDAP
     :constructed => constructed,
   }
 
+  universal = {
```

Comment:
What do you mean by 'specific value'?, to me the specific value is in the title of the PR, support for modify password http://tools.ietf.org/html/rfc3062.

What is your understanding of the mechanics of `ExtendedResponse`, if you could share with me your understanding I would be happy to make changes to this PR.

I have invested considerable time in understanding this codebase and have addressed every point raised by yourself and @jch. I have sniffed packets, groked Open LDAP source and the source of many other libs in order to comprehensively document and explain each change I have made, I am committed to delivering this feature and like I said, I am happy to address any concerns you have or explain the workings of LDAP or this library. I fully understand that you have inherited this codebase so I understand the fear you may have in accepting changes to this lib.